.github/workflows/mirror-calico.yml

```
@@ -19,7 +19,7 @@ jobs:
 
           git clone https://github.com/projectcalico/calico calico
           # get latest version
-          version=$(git -C "$_" tag | sort -V | tail -1)
+          version=$(git -C "$_" tag --sort=-authordate | head -1)
```

Do we need the latest X entries in case this job doesn't run often enough?

---

The challenge with this mirroring thing is that calico is releasing in parallel v3.21.x and v3.22.x - we currently have only interest with v3.21.x.
So we could even add the following:
```suggestion
          version=$(git -C "$_" tag --sort=-authordate --list "v3.21*" | head -1)
```

---

If we don't want to maintain this here, we should probably mirror all release tags. I would take the last 10 entries and iterate over them.

---

How about i fetch the tigera operator manifest and from there figure out what calico version is getting deployed? Let me give it a try.